README.md

```diff
@@ -9,6 +7,8 @@
 
 </div>
 
+## About clean
```

Comment:
agree with removing the centered header. for this intro, we could either remove the header entirely, or change it to just "clean". both seem cleaner to me than "About clean"

Reply:
yeah, I was thinking about that, too. You're right, let's just get rid of the header. 👍